modules/nodes-scheduler-node-selectors-configuring.adoc

```diff
@@ -29,27 +32,37 @@ spec:
 ...
 ----
 
-. Modify the master configuration file, *_/etc/origin/master/master-config.yaml_*, to add `nodeSelectorLabelBlacklist` to the `admissionConfig` section with
-the labels that are assigned to the node hosts you want to deny pod placement:
+. Create a file containing the admission controller information:
 +
+[source,yaml]
 ----
-...
-admissionConfig:
-  pluginConfig:
-    PodNodeConstraints:
-      configuration:
-        apiversion: v1
-        kind: PodNodeConstraintsConfig
-        nodeSelectorLabelBlacklist:
-          - kubernetes.io/hostname
-          - <label>
-...
+podNodeSelectorPluginConfig:
+ clusterDefaultNodeSelector: name-of-node-selector
+ namespace1: name-of-node-selector
+ namespace2: name-of-node-selector
+----
```

Comment:
Got the same question as below, is this indicating we can have multiple namespaces for the config? Can they be of the same?